### PR TITLE
BUG: to_timedelta fails on Int64 Series with null values

### LINKED ISSUE #35574: BUG: pd.to_timedelta() fails when arg is a Series with Int64 dtype
- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the latest version of pandas.

- [X] (optional) I have confirmed this bug exists on the master branch of pandas.

---

#### Code Sample, a copy-pastable example

```python
import pandas as pd


series = pd.Series([pd.NA], dtype="Int64")

pd.to_timedelta(series, errors="coerce")
```

#### Problem description

When converting a `Series` with an `Int64` dtype to a timedelta, `to_timedelta` fails with
 
```python
ValueError: cannot convert to 'int64'-dtype NumPy array with missing values. Specify an appropriate 'na_value' for this dtype.
```

Passing a Python list containing a NA value works, as does passing an `Int64` Series that does not contain NA values. The functionality between these cases should be equivalent.

#### Expected Output

`pd.to_timedelta()` output returns a `timedelta64` Series containing NA values when the input is an `Int64` Series containg NA values:

```python
0   <NA>
dtype: timedelta64[ns]
```

#### Output of ``pd.show_versions()``

<details>

INSTALLED VERSIONS
------------------
commit           : 3b6915ef9435b13ac5d8c19634270ae231f8e47c
python           : 3.7.7.final.0
python-bits      : 64
OS               : Linux
OS-release       : 5.4.0-7634-generic
Version          : #38~1595345317~20.04~a8480ad-Ubuntu SMP Wed Jul 22 15:13:45 UTC 
machine          : x86_64
processor        : 
byteorder        : little
LC_ALL           : C.UTF-8
LANG             : C.UTF-8
LOCALE           : en_US.UTF-8

pandas           : 1.2.0.dev0+28.g3b6915ef9
numpy            : 1.18.5
pytz             : 2020.1
dateutil         : 2.8.1
pip              : 20.2.1
setuptools       : 45.2.0.post20200210
Cython           : 0.29.21
pytest           : 6.0.1
hypothesis       : 5.23.11
sphinx           : 3.1.1
blosc            : None
feather          : None
xlsxwriter       : 1.3.1
lxml.etree       : 4.4.1
html5lib         : 1.1
pymysql          : None
psycopg2         : None
jinja2           : 2.11.2
IPython          : 7.17.0
pandas_datareader: None
bs4              : 4.9.1
bottleneck       : 1.3.2
fsspec           : 0.8.0
fastparquet      : 0.4.1
gcsfs            : 0.6.2
matplotlib       : 3.2.1
numexpr          : 2.7.1
odfpy            : None
openpyxl         : 3.0.4
pandas_gbq       : None
pyarrow          : 0.16.0
pytables         : None
pyxlsb           : None
s3fs             : 0.4.2
scipy            : 1.5.2
sqlalchemy       : 1.3.18
tables           : 3.6.1
tabulate         : 0.8.7
xarray           : 0.16.0
xlrd             : 1.2.0
xlwt             : 1.3.0
numba            : 0.50.1

</details>

### PR DESCRIPTION
**Pull request by egoddard**
- [X] closes #35574 
- [X] tests added / passed
- [X] passes `black pandas`
- [X] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [x] whatsnew entry

### CODE REVIEW

File: pandas/tests/tools/test_to_timedelta.py [hunk 1]
```diff
@@ -24,6 +24,18 @@ def test_to_timedelta(self):
         result = to_timedelta(Series(["1d", "1days 00:00:01"]))
         tm.assert_series_equal(result, expected)
 
+        # IntegerArray Series
+        expected = Series([timedelta(days=1), timedelta(days=2)])
+        result = to_timedelta(Series([1, 2], dtype="Int64"), unit="days")
+
+        tm.assert_series_equal(result, expected)
+
+        # IntegerArray Series with nulls
```
Comment (arw2019) [inline pandas/tests/tools/test_to_timedelta.py:33]: it looks like other tests for missing values are in 
https://github.com/pandas-dev/pandas/blob/3701a9b1bfc3ad3890c2fb1fe1974a4768f6d5f8/pandas/tests/tools/test_to_timedelta.py#L133
so might want to move this part there

**Review by arw2019 — CHANGES_REQUESTED** [review]
comment re: test placement

otherwise lgtm

File: pandas/tests/tools/test_to_timedelta.py (hunk 1 above)
Comment (jorisvandenbossche) [reply pandas/tests/tools/test_to_timedelta.py:33]: Alternatively, I would maybe just put this new code in its own test (so a test specifically for passing nullable Int64 dtype data)

**Comment by TomAugspurger** [comment]
We'll need a release note. This can go in 1.1.1 I think.

File: pandas/tests/tools/test_to_timedelta.py [hunk 2]
```diff
@@ -166,3 +166,15 @@ def test_to_timedelta_ignore_strings_unit(self):
         arr = np.array([1, 2, "error"], dtype=object)
         result = pd.to_timedelta(arr, unit="ns", errors="ignore")
         tm.assert_numpy_array_equal(result, arr)
+
+    def test_to_timedelta_nullable_int64_dtype(self):
```
Comment (jreback) [inline pandas/tests/tools/test_to_timedelta.py:170]: can you add the issue number as a comment here

**Review by jreback — CHANGES_REQUESTED** [review]

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @egoddard

**Comment by jreback** [comment]
@Meeseeksdev backport to 1.1.x

**Pull request merged**